src/components/Camera/PermissionsPrimer/assets/allow.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="275px" height="121" viewBox="0 0 275 121" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Has this already been optimised? I usually use [svgomg](https://jakearchibald.github.io/svgomg/ ) to optimise svgs.

Reply:
Great tool!